backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -218,17 +226,20 @@ def sanitize_records(self, records, selected_trees, filters):
                 archSummary["status"][status] += 1
 
             if build_id not in processed_builds:
-                processed_builds.add(build_id)
                 build = get_build(r, current_tree["index"])
+                processed_builds.add(build_id)
```

Comment:
why change this line? usually when opening a PR we want to keep only significant changes and minimize noise